src/services/loginService.ts

```diff
@@ -32,9 +34,25 @@ export class AzureLoginService {
     }
   }
 
-  public static async interactiveLogin(options: InteractiveLoginOptions): Promise<AuthResponse> {
-    await open("https://microsoft.com/devicelogin");
-    return await interactiveLoginWithAuthResponse(options);
+  public static async interactiveLogin(options?: InteractiveLoginOptions): Promise<AuthResponse> {
+    // await open("https://microsoft.com/devicelogin");
+    // return await interactiveLoginWithAuthResponse(options);
+    var authResp: AuthResponse = {credentials: undefined, subscriptions: []};
+    if(!(options.tokenCache as SimpleFileTokenCache).empty()){
```

Comment:
Can we try to use the normalized `TokenCache` interface vs. dropping down to the derived type?